float32 Angle_J1
float32 Angle_J2
float32 Angle_J3
float32 Angle_J4
float32 Angle_J5
float32 Angle_J6